### ISSUE TITLE
Implement Multi-Signature Transactions via Client CLI

### ISSUE THREAD

**Issue by AlexStroke** [hyperledger-iroha/iroha#4111]
### Requirements:

**Comment by s8sato** [comment]
- blocked by #4373

**Comment by mversic** [comment]
situation might be quite different after #4788. Do we close this ticket?

**Comment by s8sato** [comment]
I think #4788 realized multisig by combination of some existing instructions, which would require no additional commands to clients. What is your opinion @Erigara ?

**Comment by Erigara** [comment]
Yes, it is done by calling triggers, currently calling triggers is not supported by client cli, but `json` command can be used.
Also since multi-signature transactions are opt in, there is not much sense to include it into client cli.
